crime-means-assessment/src/main/resources/schemas/apiCreateMeansAssessmentRequest.json

```diff
@@ -99,6 +99,10 @@
     "type": "integer",
     "description": "Unique Submission Number - EForm Reference"
   },
+  "otherHousingNote": {

```

Comment:
This field should be within the 'properties' field of the schema, otherwise this will not be picked up. I think two other properties are also outside of the schema, (usn and assessmentType) please also move them along with the new field

Reply:
done